main.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Communet</title>
+    <link rel="stylesheet" href="style.css"> 
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"/>
+</head>
+<body>
+    <!-- home section start -->
+    <header class="home">
+        <h1>Commune</h1>
+    </header>
+    <!--about us page-->
+
+    <section class="about-us">
```

Comment:
this should be an aside and a child of the main or the header

install aXe devtools, or run lighthouse (in chrome devtools) for more automated accessibility reports